.github/workflows/cmake_win.yml

```diff
@@ -21,28 +21,48 @@ jobs:
             os: windows-2022,
             environment_script: "C:\\Program Files\\Microsoft Visual Studio\\2022\\Enterprise\\VC\\Auxiliary\\Build\\vcvars64.bat",
             generators: "Ninja",
-            build: "Debug"
+            build: "Debug",
+            openssl: true,
+            disable_openssl: "OFF",
+            testing: true
           }
         - {
             name: "Windows (VS 2022/Ninja) Release",
             os: windows-2022,
             environment_script: "C:\\Program Files\\Microsoft Visual Studio\\2022\\Enterprise\\VC\\Auxiliary\\Build\\vcvars64.bat",
             generators: "Ninja",
-            build: "Release"
+            build: "Release",
+            openssl: true,
+            disable_openssl: "OFF",
+            testing: true
           }
         - {
             name: "Windows (VS 2022/Ninja) Debug 32-bit",
             os: windows-2022,
             environment_script: "C:\\Program Files\\Microsoft Visual Studio\\2022\\Enterprise\\VC\\Auxiliary\\Build\\vcvarsamd64_x86.bat",
             generators: "Ninja",
             build: "Debug",
-            choco: "--x86"
+            openssl: true,
+            disable_openssl: "OFF",
+            choco: "--x86",
+            testing: true
+          }
+        - {
+            name: "Windows (VS 2022/Ninja) Debug ARM 64-bit",
```

Comment:
github actions truncates the name sometimes, maybe better to reorder/shorten the name:

![image](https://github.com/baresip/re/assets/296762/f5ed3483-2b73-4725-b9c3-561f193ecb31)